.pipelines/nuget_config/x64/packages.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="python" version="3.7.9" targetFramework="native" />
-  <package id="Microsoft.AI.DirectML.Preview" version="1.9.0-devb9b146539f535988728c8eb4791840db54add4a7" targetFramework="native" />
```

Comment:
How does it relate to cmake?

Reply:
as above
> fyi: github is showing strange list of commits below. As I have only a single commit that is directly on top of tag v1.12.1

Reply:
To reduce the diffs please add the change to top of 'main' not a previous release.

Reply:
I pushed only this https://github.com/diablodale/onnxruntime/tree/fix12967-removeitem
but github is attempting an unusual rebase/merge that is incorrect.

Reply:
1.12.1 was essentially forked off 'main' months ago. New changes are made to 'main'. PRs are to apply the changes to 'main' from your branch. As your branch is using 1.12.1 as its base that results in a massive diff. Github is doing what it's meant to do though. 

It might be impossible to get a simple merge of the latest code into your 1.12.1 based branch given there were a number of changes manually added to 1.12.1 for the patch release. i.e. simply doing `git merge origin/main` into your branch may result in a lot of conflicts to resolve. It may be simpler to start with a clean branch from 'main', apply your change there, and create a PR using that.

Reply:
Got it. I can do this. I'll close this PR and reopen one with the `""` fix on top of today's `main`.